### DISCUSSION TITLE
Vue3 - What is the usefulness of "Wrapper around Supabase.js"?

### DISCUSSION THREAD

**Discussion by riodw** [supabase/supabase#9485] [Questions] [answered]
_I apologize ahead of time if the answer here is simple and I just missed it._

- https://github.com/supabase-community/vue-supabase

I've seen a few of these ^ Vue wrappers for Supabase, but I have not seen an explanation as to why one would want to use them.
Can someone point me to what the differences are or even better, why using `vue-supabase` is better?

------------------

Why is doing...
```html
<script setup lang="ts">
import { useSupabase } from "vue-supabase";

const supabase = useSupabase();

const { data, error } = await supabase.from("events").select("*");
</script>
```
better than doing...
```html
<script setup lang="ts">
import { supabase } from "../supabase";

const { data, error } = await supabase.from("events").select("*");
</script>
```
?

**Comment by wobsoriano** [comment, marked as answer]
There is no real difference.

with Nuxt and https://supabase.nuxtjs.org, yes.
